server/routes/one-time-code-auth/OneTimeCodeEmailSentController.test.ts

```diff
@@ -12,15 +11,9 @@ const res = {
   redirect: jest.fn(),
 }
 
-jest.mock('../../config')
-
 describe('OneTimeCodeEmailSentController', () => {
   const oneTimeCodeEmailSentController = new OneTimeCodeEmailSentController()
 
-  beforeEach(() => {
```

Comment:
`oneTimeCodeValidityDuration` is no longer part of the view 👇 so is not needed in this test